### PR TITLE
feat: enable order book candles using feature flag

### PR DESCRIPTION
**Pull request by tinaszheng**
this pr:
- removes the ability for users to toggle on and off using the order book for candles feature
- defaults true for everyone, but behind a feature flag
- removes unused code

### CODE REVIEW

File: src/hooks/tradingView/useTradingView.ts [hunk 1]
```diff
@@ -82,6 +73,9 @@ export const useTradingView = ({
 
   const savedTvChartConfig = useAppSelector(getTvChartConfig);
   const ffEnableOrderbookCandles = useStatsigGateValue(StatsigFlags.ffEnableOhlc);
+  useEffect(() => {
```
Comment (tinaszheng) [inline src/hooks/tradingView/useTradingView.ts:76]: this line was previously in src/hooks/tradingView/useOrderbookCandles.ts

File: src/lib/tradingView/utils.ts [hunk 2]
```diff
@@ -16,7 +16,8 @@ import { AppTheme, type AppColorMode } from '@/state/appUiConfigs';
 
 import { getDisplayableTickerFromMarket } from '../assetUtils';
 
-const MIN_NUM_TRADES_FOR_ORDERBOOK_PRICES = 10;
+// Show order book candles instead of trade candles if there are no trades in that time period
+const MAX_NUM_TRADES_FOR_ORDERBOOK_PRICES = 1;
```
Comment (tinaszheng) [inline src/lib/tradingView/utils.ts:20]: i think `min` here was wrong?

File: src/lib/tradingView/utils.ts [hunk 3]
```diff
@@ -37,17 +38,17 @@ const getOhlcValues = ({
   orderbookOpen?: number;
   orderbookClose?: number;
 }) => {
-  const useOrderbookCandles =
+  const showOrderbookCandles =
     orderbookCandlesToggleOn &&
-    trades <= MIN_NUM_TRADES_FOR_ORDERBOOK_PRICES &&
+    trades <= MAX_NUM_TRADES_FOR_ORDERBOOK_PRICES &&
```
Comment (jaredvu) [inline src/lib/tradingView/utils.ts:43]: Are we opposed to displaying candles for a single trade? `<` instead of `<=`?

Comment (tinaszheng) [reply src/lib/tradingView/utils.ts:43]: i am confirming with matt and adam 🤔

Comment (tinaszheng) [reply src/lib/tradingView/utils.ts:43]: @jaredvu this is fine

**Review by jaredvu — APPROVED** [review]

**Pull request merged**